wagtail/admin/templates/wagtailadmin/login.html

```diff
@@ -64,15 +64,16 @@ <h1>{% block branding_login %}{% trans "Sign in to Wagtail" %}{% endblock %}</h1
                                     <label>
                                         <input name="remember" type="checkbox" />
                                         {% trans "Remember me" %}
-                                        {% if show_password_reset %}
-                                            <span class="help">
-                                                <a class="help__link" href="{% url 'wagtailadmin_password_reset' %}" aria-label="{% trans 'Reset your password' %}">{% trans "Forgotten password?" %}</a>
-                                            </span>
-                                        {% endif %}
                                     </label>
                                 </div>
                             </li>
-
+                            {% if show_password_reset %}
+                                <li class="help">
+                                    <div>
```

Comment:
drive-by comment: can we avoid `div`-itis like this unless strictly necessary. and if necessary, accompany it with a `{# contextual comment as to why #}` ?

Reply:
I added it initially to keep the dom structure consistent with the above content. Now, with Thibaud's PR, it has been refactored.